server/Cargo.toml

```diff
@@ -29,6 +29,9 @@ snmalloc-rs = "0.3.3"
 
 paho-mqtt = { version = "0.11.1", default-features = false, features = ["bundled"] }
 
+rusqlite = { version = "0.28.0", features = ["bundled"] }
+serde_json = "1.0"
```

Comment:
This was initially done because I was having issues with the conversion to and from bytes and JSON was the easier option. I have changed it to use a BLOB in the table and converting to and from bytes.